### DISCUSSION TITLE
Where is the examplelSite?

### DISCUSSION THREAD

**Discussion by smkplus** [hugo-fixit/FixIt#336] [Q&A] [answered]
I think it was better if you put exampleSite in theme like Love It

**Comment by Lruihao** [comment, marked as answer]
- https://github.com/hugo-fixit/hugo-fixit-blog-git
- https://github.com/hugo-fixit/hugo-fixit-blog-go
- https://github.com/hugo-fixit/docs
- https://github.com/Lruihao/hugo-blog

The example site has migrated to the submodule hugo-fixit/docs.
